src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -2643,6 +2694,18 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/InductionResponse'
+    SarContentFound:
+      description: Request successfully processed - content found
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/SarSuccessResponse'
+    SarIncorrectRequest:
```

Comment:
Rather than define a specific type for this (400 BAD REQUEST), we already have a type defined that you can reference instead 👍 
```
400ErrorResponse
```